contracts/src/OPSuccinctDisputeGame.sol

```diff
@@ -53,9 +53,10 @@ contract OPSuccinctDisputeGame is ISemver, CWIA, IDisputeGame {
     ///      i.e. The game type should indicate the security model.
     /// @return gameType_ The type of proof system being used.
     function gameType() public pure returns (GameType) {
-        // TODO: Once the following PR https://github.com/ethereum-optimism/optimism/pull/13780 is merged,
+        // TODO: Once a new version of the Optimism contracts containing the PR below is released,
```

Comment:
1. Do we need to keep the TODO even though the PR is merged?
2. Would there be a chance that OP_SUCCINCT game type randomly changes in the future?

Reply:
We need a released version of the contracts that we can use here. Don't think the type will change now that it will be upstreamed.

Reply:
Yes exactly